devices/System.Buffers.Helpers/BitConverter/EndianBitConverter.cs

```diff
@@ -0,0 +1,207 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Iot.Device.MulticastDns.Package
+{
+    /// <summary>
+    /// The <see cref="EndianBitConverter"/> class is the base class for Little and Big Endian Converters.
+    /// </summary>
+    public abstract class EndianBitConverter : IBitConverter
+    {
+        /// <summary>
+        /// Gets a static instance of a <see cref="BigEndianBitConverter"/>.
+        /// </summary>
+        public static IBitConverter Big
+        {
+            get
+            {
+                if (_big == null)
+                {
+                    _big = new BigEndianBitConverter();
+                }
+
+                return _big;
+            }
+        }
+
+        /// <summary>
+        /// Gets a static instance of a <see cref="LittleEndianBitConverter"/>
```

Comment:
Missing full stop ar sentence end. (there may be others...)

Reply:
There were indeed others, seems StyleCop doesn't pick that up on abstract classes